devops/pypi-release-new.yml

```diff
@@ -101,7 +101,7 @@ stages:
         freezeArtifact: $(freezeArtifactName)
         freezeFile: $(freezeFile)
 
-    - script: python ./scripts/build_widget.py --yarn-path /usr/bin/yarn
+    - script: python ./scripts/build_widget.py --yarn-path /usr/local/bin/yarn
```

Comment:
hadn't you fixed this already?

Reply:
In the regular pipeline. I forgot about the release pipeline :-(